Sources/GitHub/API.swift

```diff
@@ -2,64 +2,144 @@ import Foundation
 import Sugar
 
 open class API {
-	private let accessToken: String
+	typealias HTTPHeader = (field: String, value: String?)
 
-	public init(accessToken: String) {
-		self.accessToken = accessToken
+	let authorization: Authorization
+
+	init(authorization: Authorization) {
+		self.authorization = authorization
+	}
+
+	func get<ResponseModel: Codable>(
```

Comment:
<!--
  1 failure:  Prefer to locate ...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/GitHub/API.swift;
  Line: 13;
-->
- :no_entry_sign: Prefer to locate parameters with defaults toward the end of the parameter list. (`function_default_parameter_at_end`)